internal/commands/push/inputs_test.go

```diff
@@ -34,6 +34,32 @@ func TestPushInputsResolve(t *testing.T) {
 		})
 	})
 
+	t.Run("should return an error when specified local path does not exist", func(t *testing.T) {
+		profile, teardown := mock.NewProfileFromTmpDir(t, "app_init_input_test")
+		defer teardown()
+
+		localPath := "fakePath"
+
+		i := inputs{LocalPath: localPath}
+		assert.Equal(t, errProjectInvalid{Path: localPath, Exists: false}, i.Resolve(profile, nil))
+	})
+
+	t.Run("should return an error when specified local path is not a realm app project", func(t *testing.T) {
+		profile, teardown := mock.NewProfileFromTmpDir(t, "app_init_input_test")
+		defer teardown()
+
+		localPath := "pathWithNoApp"
+
+		assert.Nil(t, ioutil.WriteFile(
+			filepath.Join(profile.WorkingDirectory, localPath),
+			[]byte(`{"app_id": "eggcorn-abcde", "name":"eggcorn"}`),
```

Comment:
you probably don't need all of `WriteFile` here, if you just `os.Mkdir` to make sure the directory exists, the test should fail the same